storage/src/mmr/journaled.rs

```diff
@@ -345,8 +347,8 @@ impl<E: RStorage + Clock + Metrics, H: CHasher> Mmr<E, H> {
                 write_buffer: cfg.config.write_buffer,
                 buffer_pool: cfg.config.buffer_pool.clone(),
             },
-            cfg.lower_bound_pos,
-            cfg.upper_bound_pos,
+            cfg.lower_bound_pos.as_u64(),
```

Comment:
I guess you'll change these over to Positions when you also update the sync code to use ranges instead of start/end_pos?

Reply:
I thought it might make more sense to keep `lower_bound` and `upper_bound` as `u64` in `init_journal`.

The `fixed::Journal` doesn't know about `Location`/`Position` -- those are MMR/ADB concepts.

In this usage of `init_journal`, the bounds we're passing in are `Position`s. In `Immutable`, the bounds we pass in are `Location`s. Neither is right or wrong per se -- we're just using the underlying `u64` to index/size the journal.